---
layout: archive
title: "Research"
permalink: /research/
author_profile: true
---

{% if author.googlescholar %}
  You can also find my articles on <u><a href="{{author.googlescholar}}">my Google Scholar profile</a>.</u>
{% endif %}

{% include base_path %}

{% for post in site.publications reversed %}
  {% include archive-single.html %}
{% endfor %}

<b>Neuroscience:</b>

My recent post-doc research addressed foundational questions about the neurobiology of synaptic transmission. Synapses are fundamental building blocks of brain organization that pass messages between nerve cells, and enable networks of neurons to perform “computations” that underlie our myriad cognitive abilities. Synaptic connection strengths are also modifiable in response to specific patterns of neural activity; this process, termed ‘synaptic plasticity’, is thought to endow brains with the ability to store memories of experiences and learn new information. My work addressed specific aspects of synaptic transmission regulation at the Schaffer collateral-CA1 (SC) synapse in the adult hippocampus (a key memory center in the brains of mammals), which is associated with the encoding of recently acquired information.

On the biophysical level, synaptic plasticity is initiated when patterned activation of the synapse triggers a localized increase of calcium (Ca2+) ions inside the cell, switching on a chemical cascade of downstream molecular events. Previous studies typically have only considered the entry of calcium from outside the cell via activated (NMDA) receptor-channels on its surface. However, another potential source of calcium - intra-cellular calcium stores associated with the endoplasmic reticulum (ER) that exists around synapses - may also be engaged during the induction of synaptic plasticity. How calcium handling by the ER, a prominent multifunctional cellular organelle, contributes to this process has been unclear despite suggestive evidence from experiments. To mitigate experimental limitations and shed light on the detailed interactions among various molecular components, I developed and analyzed a physiologically grounded computational model of calcium dynamics at an individual synaptic terminal. Our [modeling](https://doi.org/10.1113/JP277726) clarifies a significant contribution of the ER to shaping calcium signals in micrometer-sized regions around the synapse, and makes specific predictions regarding its larger computational role in enabling effective and efficient use, and re-use, of resources for storing information in the pattern of synaptic connection strengths. Our work also sets up a basis for further modeling studies on the multi-scale role of the ER calcium store in neuronal functions, and resolving its potential link with the early synaptic deficits seen in Alzheimer’s disease.

Separately, I also [addressed](https://pubmed.ncbi.nlm.nih.gov/32847867/) to what extent fundamental constraints on neural design, such as information coding and local energetic efficiency, can account for some of the distinctive features of the SC excitatory synapse. This synapse is marked by unreliable transmission of presynaptic spikes (via stochastic release of neurotransmitter-laden vesicles), and shows short-lived plasticity (short-term facilitation on timescales of < 1 sec) that is associated with some transient forms of memory (‘working memory’). Our computational analysis indicates a novel trade-off between performance and metabolic cost that is distinct from other synapse types in sensory pathways, and suggests that this synapse operates within a range that supports optimal transmission of information while keeping energy needs low.

<b>Summary of previous work in bioinformatics:</b>

<b>Relating gene expression changes to upstream regulatory activity ([Paper + code](https://doi.org/10.1371/journal.pone.0142147)):</b> 

Context: Functional genomics experiments such as microarrays and RNA-seq yield large scale profiles of gene expression changes between conditions, reflecting cellular reorganization during adaptation, differentiation, or disease. A key step in the analysis and interpretation of such profiles is to infer the upstream regulators (transcription factors/TFs) whose differential activity can best account for these changes. Existing approaches typically apply statistical enrichment tests to rank TFs according to the association of their target gene set with the differentially expressed genes. Such methods, which evaluate TFs in an individual manner, may be inadequate or lead to spurious inferences when transcriptomic changes arise from concurrent up/down regulation of activity of multiple TFs.
<br><br>Research contribution: We proposed a generalized computational approach to identify such a set of differentially active TFs, by combining differential expression profiles with pre-assembled TF regulatory networks. Conceptually, our method identifies the optimal subset of TFs with maximal collective association of their targets with the differentially regulated gene set, in contrast to testing TFs one at a time. As a combinatorially large number of possible TF groupings precludes ranking-based identification of the optimal subset, we proposed and evaluated greedy methods to find a ‘good’ solution. These approximate methods (particularly suitable for large meta-analyses) were benchmarked with respect to random search and stochastic (simulated annealing-based) methods on microarray expression profiles for <i>E. coli</i>. Our analysis demonstrates improved recovery of candidate regulators with our approach when applied to previously published <i>E. coli</i>. and <i>M. tuberculosis</i> genome-wide expression datasets, as well as on simulated differential expression profiles, and in particular, it mitigates bias in inference for TFs with larger target gene sets. Our approach should be a valuable contribution to gene expression analysis workflows, as it provides a crucial step to link experimental observations to biologically meaningful hypotheses about the underlying causative molecular pathways. The method may be extended to account for TF-TF interactions (co-regulation), or have scoring functions that also account for additional details, such as TF-promoter binding affinities.
<br>

<b>Host-pathogen protein interactome analysis ([Paper](https://doi.org/10.1186/s12859-017-1550-y)):</b> 

Context: Mapping of the protein interactome between host and pathogen proteomes is an important goal toward understanding the molecular basis of infection and disease progression, as well identifying potential therapeutic targets. Computational prediction of protein-protein interactions (PPIs) can complement and contribute to experimental studies toward this end. Several existing prediction methods rely on sequence-based analysis of similarities between proteins (sequence homology) and existing PPI information. However, both local and global sequence alignment, basis for such methods, imply limitations on accurate identification of novel candidate interactions, and may lead to either substantial proportion of false positives or miss out on potential interactions.
<br><br>Research contribution: We systematically evaluated the limitations of local sequence alignment for inference of candidate PPIs. To mitigate these caveats we proposed a data-integration approach to prioritize potential host-pathogen protein-protein interactions that combines structure with sequence information. Our approach relies on experimentally identified interacting regions in existing Protein Data Bank (PDB) protein complexes, and these serve as templates for targeted scoring of potential interactions between host and microbe proteins. This strategy reduces chances of spurious predictions. We applied our methodology to identify novel candidate interactions for the human pathogen <i>M. tuberculosis</i>. To expand the possible search space, information from PDB was integrated with existing data on conserved interactions (‘interologs’ of template proteins). The prioritized PPI candidates were then filtered by combining with protein metadata (cellular localization, disease association, functional annotations, etc.) to rank them according to their plausibility and evaluate their functional relevance. Our bioinformatics analysis underlines limitations of existing sequence alignment and domain-based PPI search methods. Altogether, our approach provides a more informed application of sequence alignment toward an efficient, scalable and high-specificity method for protein interaction prediction between species based on “sequence mimicry”, by leveraging existing PPI and structural data as priors. More recently, we have extended our approach to include structure-based protein descriptors for improved predictions, esp. to better handle structural mimicry (evolutionary convergence with little sequence similarity) in protein comparisons.
<br>
